gremlin-core/src/test/java/org/apache/tinkerpop/gremlin/language/grammar/TraversalMethodVisitorTest.java

```diff
@@ -696,6 +700,30 @@ public void testTraversalMethod_property_Object_Object_Object() throws Exception
         compare(g.V().property(1,2,"key", 4), eval("g.V().property(1,2,'key',4)"));
     }
 
+    @Test
+    public void testTraversalMethod_property_Object() throws Exception {
+        LinkedHashMap<Object, Object> map = new LinkedHashMap<>();
```

Comment:
can this work without a `LinkedHashMap` here? i think it would be preferable to test with a regular old `HashMap`.

Reply:
If you test with a regular HashMap you have no guarantee that the items will appear in the Bytecode in the same order as the eval() statement.  It will compile and run but randomly fail do to the non-deterministic aspects.

Reply:
ok, so the `LinkedHashMap` is for testing purposes